### DISCUSSION TITLE
Why is the cooldown on .admin THAT high

### DISCUSSION THREAD

**Discussion by Gamer086** [codeman1o1/TIJK-Bot#188] [Q&A] [answered]
The cool-down is 100 sec, shouldnt it be like 10 or something

**Comment by karstenzip** [comment, marked as answer]
Actually the cooldown is 120 seconds (2 minutes) and for the urgent version it is 300 seconds (5 minutes).
This is fully intended as it is not ment to be used frequently and it could serve as a spam method. **IF YOU USE `.admin (urgent)` AS A SPAM METHOD THE COMMAND WILL BE DISABLED AND YOU WILL BE MUTED OR WORSE!!!**

**Comment by Gamer086** [comment]
Im not talking about spam…
